types/aws-api-gateway-client/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Why this config change? See: https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson

Reply:
Removed the rule.